octomap_server/include/octomap_server/OctomapServer.h

```diff
@@ -218,6 +218,7 @@ class OctomapServer {
   std_msgs::ColorRGBA m_color;
   std_msgs::ColorRGBA m_colorFree;
   double m_colorFactor;
+  std::string m_map_file;
 
```

Comment:
I don't think there's a need for a member variable to store the map file. It is only used in the constructor so it should be a local variable there (see `probHit`)